docs/redirects.yml

```
@@ -1,4 +1,3 @@
 redirects:
-  # Contributing to docs
   'extend/development-documentation.md': 'docs-content://extend/contribute/index.md'
-  'extend/saved-objects-service.md': 'docs-content://extend/saved-objects.md'
+  'extend/saved-objects-service.md': 'extend/saved-objects.md'
```

The redirects feature is not working as expected.

To make this work we need mock the actual url


```suggestion
  'extend/kibana/saved-objects-service.md': 'extend/saved-objects.md'
```